paymentsheet/src/main/java/com/stripe/android/paymentsheet/specifications/ResourceRepository.kt

```diff
@@ -0,0 +1,16 @@
+package com.stripe.android.paymentsheet.specifications
+
+import com.stripe.android.paymentsheet.address.AddressFieldElementRepository
+
+/**
+ * This holds all the resources read in from JSON.
+ */
+internal class ResourceRepository(
```

Comment:
Same here, since both `BankRepository` and `AddressFieldElementRepository` are already provided, you can just change it to the following and remove the method from `Module`

```
@Singleton
internal class ResourceRepository @Inject internal constructor(
    internal val bankRepository: BankRepository,
    internal val addressRepository: AddressFieldElementRepository
)
```